src/drivers/localstorage.js

```diff
@@ -24,6 +24,14 @@ function _initStorage(options) {
         dbInfo.keyPrefix += dbInfo.storeName + '/';
     }
 
+    try {
+        var localStorageTestKey = dbInfo.keyPrefix + 'check';
```

Comment:
How about abstracting this into a new function and using a different prefix like in indexeddb?
eg `_localforage_support_test`